sycl/include/sycl/detail/pi.h

```diff
@@ -434,7 +436,9 @@ typedef enum {
   PI_COMMAND_TYPE_SVM_MEMCPY = 0x120A,
   PI_COMMAND_TYPE_SVM_MEMFILL = 0x120B,
   PI_COMMAND_TYPE_SVM_MAP = 0x120C,
-  PI_COMMAND_TYPE_SVM_UNMAP = 0x120D
+  PI_COMMAND_TYPE_SVM_UNMAP = 0x120D,
+  PI_COMMAND_TYPE_DEVICE_GLOBAL_VARIABLE_READ = 0x418E,
+  PI_COMMAND_TYPE_DEVICE_GLOBAL_VARIABLE_WRITE = 0x418F
```

Comment:
You add/set these types, but never read/rely on them. In this case we were just using generic `PI_COMMAND_TYPE_USER`. This is grey area, so if you still prefer to add these then please use PIEXT_  prefix and document what they mean.

Reply:
They correspond to the command types in the [cl_intel_global_variable_access](https://github.com/intel/llvm/blob/sycl/sycl/doc/design/opencl-extensions/cl_intel_global_variable_access.asciidoc) (`CL_COMMAND_READ_GLOBAL_VARIABLE_INTEL` and `CL_COMMAND_WRITE_GLOBAL_VARIABLE_INTEL` respectively) extension, so I worry `PIEXT_` may incorrectly give the impression that this is a PI-only command type. Do we normally add `PIEXT_` for OpenCL extension values?

Reply:
I didn't realize they are in OpenCL, then no PIEXT_ is needed.